{"name": "TC#1 - vwo Login testcase", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Timeout 30000ms exceeded.", "trace": "self = <tests_vwo_login.TestLogin object at 0x111a2f510>\npage = <Page url='https://app.vwo.com/#/login'>\n\n    @allure.title(\"TC#1 - vwo Login testcase\")\n    def test_valid_login(self, page):\n        login_page = Login(page)\n        dashboard_page = Dashboard(page)\n>       login_page.navigate()\n\ntests_vwo_login.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n../pages/login_page.py:31: in navigate\n    self.page.goto(\"https://app.vwo.com\")\n../../../venv/lib/python3.11/site-packages/playwright/sync_api/_generated.py:9312: in goto\n    self._sync(\n../../../venv/lib/python3.11/site-packages/playwright/_impl/_page.py:475: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n../../../venv/lib/python3.11/site-packages/playwright/_impl/_frame.py:139: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n../../../venv/lib/python3.11/site-packages/playwright/_impl/_connection.py:62: in send\n    return await self._connection.wrap_api_call(\n../../../venv/lib/python3.11/site-packages/playwright/_impl/_connection.py:492: in wrap_api_call\n    return await cb()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <playwright._impl._connection.Channel object at 0x111a07b90>\nmethod = 'goto', params = {'url': 'https://app.vwo.com'}, return_as_dict = False\n\n    async def inner_send(\n        self, method: str, params: Optional[Dict], return_as_dict: bool\n    ) -> Any:\n        if params is None:\n            params = {}\n        callback = self._connection._send_message_to_server(\n            self._object, method, params\n        )\n        if self._connection._error:\n            error = self._connection._error\n            self._connection._error = None\n            raise error\n        done, _ = await asyncio.wait(\n            {\n                self._connection._transport.on_error_future,\n                callback.future,\n            },\n            return_when=asyncio.FIRST_COMPLETED,\n        )\n        if not callback.future.done():\n            callback.future.cancel()\n>       result = next(iter(done)).result()\nE       playwright._impl._errors.TimeoutError: Timeout 30000ms exceeded.\n\n../../../venv/lib/python3.11/site-packages/playwright/_impl/_connection.py:100: TimeoutError"}, "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1707988329079, "stop": 1707988361339, "uuid": "de67c4b5-eeed-46c6-a4ae-b7f7538f4a17", "historyId": "e7a73ec1f74f8bf6498a1dae77b3730a", "testCaseId": "657f2097de3e383a16c160a0555f8ed4", "fullName": "code.pom.tests.tests_vwo_login.TestLogin#test_valid_login", "labels": [{"name": "parentSuite", "value": "code.pom.tests"}, {"name": "suite", "value": "tests_vwo_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "MBP-C02DRR20MD6M"}, {"name": "thread", "value": "39437-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "code.pom.tests.tests_vwo_login"}]}